zhaquirks/ikea/starkvind.py

```diff
@@ -220,3 +219,90 @@ def __init__(self, *args, **kwargs):
             },
         },
     }
+
+
+class IkeaSTARKVIND_v2(CustomDevice):
+    """STARKVIND Air purifier by IKEA of Sweden."""
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        self.pm25_bus = Bus()
+        self.change_fan_mode_bus = Bus()
+        self.change_fan_mode_ha_bus = Bus()
+        super().__init__(*args, **kwargs)
```

Comment:
I think it might be a good idea if ``IkeaSTARKVIND_v2`` would extend the original ``IkeaSTARKVIND`` quirk class instead of ``CustomDevice``.
Then, the ``__init__`` method could be removed in the v2 class, so it isn't duplicated in the quirk file.

Reply:
+1

Reply:
Done